src/Views/NightLightView.vala

```diff
@@ -26,6 +26,15 @@ public class Display.NightLightView : Gtk.Grid {
         }
     }
 
+    public bool active {
```

Comment:
I don't think that really is a good solution to just make a new property that's only settable + the fact that it only can receive `true`'s is not great. I would just go for a simple `changed` callback on the switch rather than this.